### PR TITLE
Add Good First Issue template

### PR DESCRIPTION
**Pull request by p-wysocki**
### Details:
 - Add template for developers who want to create a Good First Issue
 - Please add suggestions in the comments
 - The template is currently loosely based on existing GFIs, such as https://github.com/openvinotoolkit/openvino/issues/17501

### Tickets:
 - N/A

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/good_first_issue.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    validations:
+      required: true
+
+  - type: textarea
+    id: contact_points
+    attributes:
+      label: Contact points
+      description: |
+          People who can be asked questions about the task.
+      placeholder: @openvino-pushbot
```
Comment (mlukasze) [inline .github/ISSUE_TEMPLATE/good_first_issue.yml:57]: is it correct yaml syntax? linter shows it as a bug

Comment (p-wysocki) [reply .github/ISSUE_TEMPLATE/good_first_issue.yml]: Good point, apparently yaml doesn't like `@`. Fixed.

**Review by mlukasze — APPROVED** [review]

**Review by ilya-lavrenov — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/good_first_issue.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      label: What needs to be done?
+      description: |
+          Be as verbose as possible, provide a TODO list if viable.
+    validations:
+      required: true
+
+  - type: textarea
+    id: example_prs
+    attributes:
+      label: Exmaple Pull Requests
```
Comment (ilya-lavrenov) [inline .github/ISSUE_TEMPLATE/good_first_issue.yml:30]: Example

Comment (p-wysocki) [reply .github/ISSUE_TEMPLATE/good_first_issue.yml:30]: Good catch https://github.com/openvinotoolkit/openvino/pull/19898

**Pull request merged**